backend/src/player/player.controller.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Body,
+    Patch,
+    Param,
+    Delete,
+    UsePipes,
+    ValidationPipe,
+} from '@nestjs/common'
+import { PlayerService } from './player.service'
+import { CreatePlayerDto } from './dto/create-player.dto'
+
+@Controller('player')
+export class PlayerController {
+    constructor(private readonly playerService: PlayerService) {}
+
+    @Post()
+    @UsePipes(ValidationPipe)
+    async create(@Body() createPlayerDto: CreatePlayerDto) {
+        console.log(createPlayerDto)
+        const player = this.playerService.create(createPlayerDto)
+        return player
+    }
+
+    @Get()
+    async findAll() {
+        const player = await this.playerService.findAll()
```

Comment:
Should we be more explicit with types ?
```suggestion
    async findAll(): Promise<Player[]> {
        const player: Player[] = await this.playerService.findAll()
```
without forget to import `Player` with
```ts
import { Player } from 'src/typeorm/Player'
```

Reply:
I have been reading some articles about that:
https://dev.to/pffigueiredo/typescript-return-types-separating-fact-from-fiction-4266
https://www.totaltypescript.com/tips/dont-use-return-types-unless

It's a choice, I think that in this case it's pretty clear:
async findAll() in class Player return a Promise <Player[]>

Reply:
Interesting, thank you for the resources